docs/archive-data.md

```diff
@@ -136,58 +100,60 @@ namespaceDefaults:
       state: "disabled"
 ```
 
-:::
-
-### Namespace creation
+The following table contains acceptable values for each configuration and what they do. 
 
-The Archival URI is set when a Namespace is created. If you do not specify the URI when creating a Namespace, it will use the value of `defaultNamespace.archival.history.URI` from the `config/development.yaml` file. Once set, the URI can not be changed. Each Namespace supports a single Archival URI, but each Namespace can use a different URI (archiver).
+| Config | Acceptable values | Description |
+|--------|-------------------|-------------|
+| `archival.history.state`                       | `enabled`, `disabled`  | This must be set to `enabled` to use the Archival feature with any Namespace in the cluster. |
```

Comment:
I think these could be less verbose

"This must be set to `enabled` to use the Archival feature with any Namespace in the cluster."

Could be:

"Must be `enabled` to use the Archival feature with any Namespace."

Reply:
Same is true for other Descriptions.